app/api/sul_bib/authorship_api.rb

```diff
@@ -26,8 +24,8 @@ def create_or_update_and_return_pub_hash(pub, author, authorship)
         contrib = pub.contributions.find_or_initialize_by(author_id: author.id)
         contrib.assign_attributes(authorship.merge(cap_profile_id: author.cap_profile_id, author_id: author.id))
         pub.pubhash_needs_update! if contrib.persisted?
-        contrib.save
         begin
+          contrib.save!
```

Comment:
Does this change entail adding a spec for the exception handing or logging?  Maybe not, as it just appears like a response failure anyway, I dunno.

Reply:
The point was to move the contrib.save inside the existing exception trapping block and make it required (add `!`).  I didn't want `contrib.save` to be able to silently fail.